docs/conf.py

```diff
@@ -0,0 +1,76 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'vsutil'
+copyright = '2020, Irrational Encoding Wizardry'
+author = 'Irrational Encoding Wizardry'
+
+# The full version, including alpha/beta/rc tags
+version = release = '0.5.0'
```

Comment:
Is this going to be another place I need to remember updating whenever we release a new version?

Reply:
What about bumpversion?

Reply:
It's interpreted Python code, so you could read/load it from somewhere.

Reply:
So, I tried to do this all with an import from a `__version__` dunder in the main module, (it would be here and in setup.py and for anywhere else we needed it in the future) but it doesn't work, the package gets like halfway initialized and then says something like unknown relative import with no parent package or something like that. Unless I was doing something wrong, I think sadly yes, we'd now have two spots where we have to bump the version.

Reply:
So you don't have to google: https://pypi.org/project/bump2version/

This project can automate bumping the version strings as they tend to be repeated at a lot of places.

Reply:
Seems overengineered to me. `sed 's/0\.5\.0/0.6.0/g` should work just as well.

Reply:
Or a python regexp, but bump2version seems like a useful tool regardless, especially if there is already more than one location where it needs to be updated.